jwst/ami/ami_analyze.py

```diff
@@ -11,7 +11,7 @@
 from . import webb_psf
 from . import leastsqnrm
 
-log = logging.getLogger(__name__)
+log = logging.getLogger('.'.join(['stpipe', __name__]))
```

Comment:
Should this be some function provided by `jwst.stpipe.log`? That would centralize the concept that this is some `stpipe`-related thing.

Reply:
Yes! 

That was close to what I had in mind for one of my solutions. I had made the modification in `stpipe.log`, but what was happening was using the function defined there throughout the project occasionally caused import errors (circular imports) which required either refactoring of imports or refactoring of modules, so in case this needed to be fixed ASAP, this was a way to get the same result.